lego/apps/events/models.py

```diff
@@ -86,6 +87,7 @@ def registration_close_time(self):
     youtube_url = CharField(
         max_length=200, default="", validators=[youtube_validator], blank=True
     )
+    share_info_flag = models.BooleanField(default=False)
```

Comment:
Have we settled on `share_info_flag`? This is essentially _only_ for smittesporing, and not a general `share-info`-flag

Reply:
What about `share_for_conact_tracing` or something along those lines?

Reply:
maybe `use_contact_tracing`?